scipy-1.0/paper.tex

```diff
@@ -1004,8 +1005,8 @@ \subsubsection*{Benchmark suite}
 Any pull request can be compared against the \texttt{master} branch with the
 command \texttt{asv continuous master new-feature}. This will provide a
 benchmark against the master branch and the branch a new feature is implemented
-in. More features are available in the documentation, including arguments to
-select which benchmarks to run.
+in. More features are available in the \texttt{asv} documentation, including 
```

Comment:
Even though asv has already been cited, I think it would be appropriate to cite the asv documentation itself here.
https://asv.readthedocs.io/en/stable/index.html

Reply:
done